syntax = "proto3";

service Bartender {
    // make a drink
    rpc MakeDrink (DrinkRequest) returns (DrinkResponse);
}

// request message containing the drink
message DrinkRequest{
    string drink = 1;
}

// response containing the status
message DrinkResponse {
    bool success = 1;
    string message = 2;
}